shell_integration/windows/OCContextMenu/OCClientInterface.cpp

```diff
@@ -28,17 +28,22 @@
 #include <iterator>
 #include <unordered_set>
 
-// use std::min in gdiplus
-using namespace std;
+// gdiplus min/max
+#define min std::min
```

Comment:
Why not `using std::min` and `using std::max`? You put `using namespace std` below anyway, which has the same effect.

Reply:
If I use std from the beginning we get a clash in a legacy windows header with `struct byte` and std::byte or what ever crazyness.

Reply:
using std::min is much better thx, when handling such windows stuff I kind of start thinking in macros and defines...